packages/web-app-files/src/components/SideBar/Shares/SpaceShares.vue

```diff
@@ -87,6 +87,16 @@ export default {
       return currentUserCollaborator?.role?.name === spaceRoleManager.name
     }
   },
+  watch: {
+    highlightedFile: {
+      handler: function (newItem, oldItem) {
+        if (oldItem !== newItem) {
+          this.loadSharesTask.perform(this)
+        }
+      },
+      immediate: true
+    }
+  },
   mounted() {
     this.loadSharesTask.perform(this)
```

Comment:
with `immediate: true` you should be able to remove the loading task from the mounted hook (and with that the delete the mounted hook entirely).

Reply:
Indeed, thx!